extra/resources/ifspeed

```diff
@@ -165,13 +182,36 @@ monitor() {
     return ${ret}
 }
 
+# This function tries to guess nic interface by IP
+get_nic_name_by_ip(){
+    # $FINDIF takes its parameters from the environment
+    NICINFO=`${FINDIF}`
```

Comment:
Could you please declare NICINFO as local and replace subshell syntax with
NICINFO=$( ${FINDIF} )

Reply:
Why? What is the problem with subshell syntax?

Reply:
There's not a problem per se, but the dollar-parentheses syntax has some advantages:
* It's the POSIX standard; backticks are a historical syntax.
* It's easier to read when the command being executed has single and/or double quotes.
* It's easier to nest commands, e.g. V=$(echo hello $(echo world)) vs V=`echo hello \`echo world\``

Reply:
OK - I will change that. Thanks for the detailed explanation.